lib/resources/r4/family_member_history.yaml

```diff
@@ -1,28 +1,554 @@
+          {
+            "code": "male",
+            "system": "http://hl7.org/fhir/administrative-gender"
+          }
+        ]
+      }
+    }
+  binding:
+    description: The gender of a person used for administrative purposes.
+    terminology:
+      - display: AdministrativeGender
+        system: http://hl7.org/fhir/administrative-gender
+        info_link: https://hl7.org/fhir/valueset-administrative-gender.html
+
+- name: born[x]
+  type: date
+  required: 'No'
+  description: The actual or approximate date of birth of the relative.
+  url: https://hl7.org/fhir/familymemberhistory-definitions.html#FamilyMemberHistory.born_x_
+  action:
+    - create
+    - update
+  example: |
+    {
+      "bornDate": "1998-12-07"
+    }
+
+- name: deceased[x]
+  type: Boolean, Quantity
+  required: 'No'
+  description: Deceased flag or the approximate age of the relative at the time of death for the family member history record.
+  url: https://hl7.org/fhir/familymemberhistory-definitions.html#FamilyMemberHistory.deceased_x_
+  action:
+    - create
+    - update
+  note: |
+    <ul>
+      <li>When deceased field is not provided, it is defaulted to deceasedBoolean false. </li>
+    </ul>
+  example: |
+    {
+      "deceasedAge": {
+        "value": 42,
+        "system": "http://unitsofmeasure.org",
+        "code": "a",
+        "extension": [
+          {
+            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/precision",
+            "valueCodeableConcept": {
+              "coding": [
+                {
+                  "code": "26175008",
+                  "system": "http://snomed.info/sct"
+                }
+              ]
+            }
+          }
+        ]
       }
-    note: resourceType must be FamilyMemberHistory
-
-  - name: id
-    required: 'Yes'
-    type: id
-    url: https://www.hl7.org/fhir/r4/resource-definitions.html#Resource.id
-    description: The logical id of the resource.
-    action: update
-    example: |
-      {
-        "id": "123-456"
-      }
\ No newline at end of file
+    }
+  example2: |
+    {
+      "deceasedBoolean": true
+    }
+
+- name: deceasedAge.unit
+  type: Quantity
+  description: Approximate age of the relative at the time of death for the family member history record.
+  url: https://hl7.org/fhir/familymemberhistory-definitions.html#FamilyMemberHistory.deceased_x_
+  action: terminology
+  binding:
+    description: Code system intended to include all units of measures.
+    terminology:
+      - display: Units of Measure
+        system: http://unitsofmeasure.org
+        info_link: https://ucum.org/trac
+
+- name: deceasedAge.extension(precision)
+  type: CodeableConcept
+  required: 'Yes'
+  description: Indication of the precision of a given value.
+  url: https://hl7.org/fhir/extensibility.html#Extension
+  action:
+    - create
+    - update
+    - terminology
+  note: |
+    <ul>
+      <li> Supported codes include 397669002, 26175008, 288563008, 272113006 </li>
```

Comment:
Do we want to specify what these codes stand for?

Reply:
Hmm...I don't think we should specify which codes are supported. Let's just list that only snomed codes are supported. We can be more particular if needed in the future.

Reply:
removed https://github.com/cerner/fhir.cerner.com/pull/728/commits/2e9e8da0d0db2c5390c8c1fed4efe0d224b4b043#diff-784415f7640df65b77af88df40ea89d5c05f009fbe1d256c1c8a0702baa87d63L255